Cargo.toml

```diff
@@ -87,8 +88,8 @@ http1 = []
 http2 = ["h2"]
 
 # Client/Server
-client = []
-server = []
+client = ["dep:smallvec"]
+server = ["dep:smallvec"]
```

Comment:
I just ported it from https://github.com/hyperium/hyper/pull/3523. 
I'm not sure if it's required or not.